boards/lilygo/ttgo_t8s3/ttgo_t8s3_procpu.dts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32s3/esp32s3_mini_n8.dtsi>
```

Comment:
nit: the board is equipped with a 16MB flash

Reply:
Does the esp32s3_mini_n8 refer to 8MB?

The documentation mentions 16MB:

Functional Description 
**********************
This board is based on the ESP32-S3 with 16MB of flash, WiFi and BLE support....

Reply:
Same here:
Without 8MB PSRAM: `esp32s3_wroom_n16.dtsi`
With PSRAM: `esp32s3_wroom_n16r8.dtsi`